Cargo.lock

```diff
@@ -93,7 +93,7 @@ dependencies = [
  "bitflags 1.3.2",
  "bytes",
  "futures-util",
- "http",
+ "http 0.2.9",
```

Comment:
We'll now have 2 versions of the `http` crate in our dependencies. I'm not against, this, but I'd like to double check that you're aware of this before merging.